hw1/src/main/java/com/example/hw1/MovieRatingRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.example.hw1;
+
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+
+public interface MovieRatingRepository extends CrudRepository<MovieRating, Long> {
+    List<MovieRating> findByOrderByDateDesc();
```

Comment:
I do not see a @Query being used for this. The professor showed us how to create a specific query for the movie, and he also used it on his files for the person project.

Reply:
Added, thanks